### PR TITLE
Mpt 39: make mpt looks like other tests

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

File: mpt/core/src/main/java/org/apache/james/mpt/script/ImapScriptedTestProtocol.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package org.apache.james.mpt.script;
```
Comment (aduprat) [inline mpt/core/src/main/java/org/apache/james/mpt/script/ImapScriptedTestProtocol.java]: license missing

File: mpt/core/src/main/java/org/apache/james/mpt/script/SimpleScriptedTestProtocol.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package org.apache.james.mpt.script;
```
Comment (aduprat) [inline mpt/core/src/main/java/org/apache/james/mpt/script/SimpleScriptedTestProtocol.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraAuthenticatePlain.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraAuthenticatePlain.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraAuthenticatedState.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraAuthenticatedState.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraConcurrentSessions.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraConcurrentSessions.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraCondstore.java [hunk 6]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraCondstore.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraCopy.java [hunk 7]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraCopy.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraEvents.java [hunk 8]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraEvents.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraExpunge.java [hunk 9]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraExpunge.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraFetch.java [hunk 10]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraFetch.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraFetchBodySection.java [hunk 11]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraFetchBodySection.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraFetchBodyStructure.java [hunk 12]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraFetchBodyStructure.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraFetchHeaders.java [hunk 13]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraFetchHeaders.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraListing.java [hunk 14]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraListing.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraMailboxAnnotation.java [hunk 15]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraMailboxAnnotation.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraMailboxWithLongNameError.java [hunk 16]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraMailboxWithLongNameError.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraMove.java [hunk 17]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraMove.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraNonAuthenticatedState.java [hunk 18]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraNonAuthenticatedState.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraPartialFetch.java [hunk 19]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraPartialFetch.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraQuotaTest.java [hunk 20]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraQuotaTest.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraRename.java [hunk 21]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraRename.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraSearch.java [hunk 22]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraSearch.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraSecurity.java [hunk 23]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraSecurity.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraSelect.java [hunk 24]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraSelect.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraSelectedInbox.java [hunk 25]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraSelectedInbox.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraSelectedState.java [hunk 26]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraSelectedState.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraUserFlagsSupport.java [hunk 27]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraUserFlagsSupport.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraUidSearchOnIndex.java [hunk 28]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraUidSearchOnIndex.java]: license missing

File: mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraUidSearch.java [hunk 29]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.cassandra;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cassandra/src/test/java/org/apache/james/mpt/imapmailbox/cassandra/CassandraUidSearch.java]: license missing

File: mpt/impl/imap-mailbox/core/src/main/java/org/apache/james/mpt/imapmailbox/suite/ACLScriptedTestProtocol.java [hunk 30]
```diff
@@ -0,0 +1,82 @@
+package org.apache.james.mpt.imapmailbox.suite;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/core/src/main/java/org/apache/james/mpt/imapmailbox/suite/ACLScriptedTestProtocol.java]: license missing

File: mpt/impl/imap-mailbox/cyrus/src/test/java/org/apache/james/mpt/imapmailbox/cyrus/CyrusACLCommands.java [hunk 31]
```diff
@@ -0,0 +1,41 @@
+package org.apache.james.mpt.imapmailbox.cyrus;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cyrus/src/test/java/org/apache/james/mpt/imapmailbox/cyrus/CyrusACLCommands.java]: license missing

File: mpt/impl/imap-mailbox/cyrus/src/test/java/org/apache/james/mpt/imapmailbox/cyrus/CyrusACLIntegration.java [hunk 32]
```diff
@@ -0,0 +1,49 @@
+package org.apache.james.mpt.imapmailbox.cyrus;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/cyrus/src/test/java/org/apache/james/mpt/imapmailbox/cyrus/CyrusACLIntegration.java]: license missing

File: mpt/impl/imap-mailbox/elasticsearch/src/test/java/org/apache/james/mpt/imapmailbox/elasticsearch/ElasticSearchUidSearchOnIndex.java [hunk 33]
```diff
@@ -0,0 +1,33 @@
+package org.apache.james.mpt.imapmailbox.elasticsearch;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/elasticsearch/src/test/java/org/apache/james/mpt/imapmailbox/elasticsearch/ElasticSearchUidSearchOnIndex.java]: license missing

File: mpt/impl/imap-mailbox/external-james/src/test/java/org/apache/james/mpt/imapmailbox/external/james/JamesDeploymentValidation.java [hunk 34]
```diff
@@ -0,0 +1,34 @@
+package org.apache.james.mpt.imapmailbox.external.james;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/external-james/src/test/java/org/apache/james/mpt/imapmailbox/external/james/JamesDeploymentValidation.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseAuthenticatePlain.java [hunk 35]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseAuthenticatePlain.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseAuthenticatedState.java [hunk 36]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseAuthenticatedState.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseConcurrentSessions.java [hunk 37]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseConcurrentSessions.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseCondstore.java [hunk 38]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseCondstore.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseCopy.java [hunk 39]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseCopy.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseEvents.java [hunk 40]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseEvents.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseExpunge.java [hunk 41]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseExpunge.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseFetch.java [hunk 42]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseFetch.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseFetchBodySection.java [hunk 43]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseFetchBodySection.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseFetchBodyStructure.java [hunk 44]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseFetchBodyStructure.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseFetchHeaders.java [hunk 45]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseFetchHeaders.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseListing.java [hunk 46]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseListing.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseMailboxAnnotation.java [hunk 47]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseMailboxAnnotation.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseMailboxWithLongNameError.java [hunk 48]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseMailboxWithLongNameError.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseMove.java [hunk 49]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseMove.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseNonAuthenticatedState.java [hunk 50]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseNonAuthenticatedState.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBasePartialFetch.java [hunk 51]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBasePartialFetch.java]: license missing

File: mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseQuotaTest.java [hunk 52]
```diff
@@ -0,0 +1,32 @@
+package org.apache.james.mpt.imapmailbox.hbase;
```
Comment (aduprat) [inline mpt/impl/imap-mailbox/hbase/src/test/java/org/apache/james/mpt/imapmailbox/hbase/HBaseQuotaTest.java]: license missing

**Comment by chibenwa** [comment]
I have a commit somewhere for those licenses

**Pull request merged**